faq.md

```diff
@@ -61,6 +88,15 @@ Older versions of Safari on iPhones, iPads, and Safari on macOS on M1 (ARM) mach
 
 Newer updates of Safari no longer have this issue, but some older [devices may need to use a workaround](running/safari.html).
 
+##### Blank page with incorrect time
+
+A blank page after logging in also may be a symptom of a wildly incorrect system time on your server, as all certficiates have a start and end date.
```

Comment:
This sounds very unplausible to me, do you have a link where this happened? The TLS certificates are evaluated by the browser, not by the server. And if they expire, they are simply invalid. In neither case they appear as blank page, but the browser won't let you visit cockpit at all.

Reply:
Yes, it happened. https://github.com/cockpit-project/cockpit/issues/17995

Reply:
But it's not clear whether the wrong time happened on the server or the client, and what browser was used. Why would the browser accept an invalid certificate for the html/css pages, but not for the websocket?  This wasn't asked "frequently", not debugged sufficiently, and sounds very dubious to me. IMHO we shouldn't duplicate all weird corner cases, otherwise the FAQ would both become really long and filled with dubious stuff, an also not really be "F".

I converted that issue into a discussion https://github.com/cockpit-project/cockpit/discussions/18060 and asked the reporter about some details.